io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,185 @@
+
+- name: qscintilla
+  buildsystem: qmake
+  builddir: true
+  subdir: Qt4Qt5
+  config-opts:
+    - CONFIG+=release  
+  sources:
+    - type: archive
+      url: https://www.riverbankcomputing.com/static/Downloads/QScintilla/2.11.4/QScintilla-2.11.4.tar.gz
+      sha256: 723f8f1d1686d9fc8f204cd855347e984322dd5cd727891d324d0d7d187bee20
+    - type: shell
+      commands:
+        - sed -e 's|\(target.path\) = .*|\1 = /app/lib|'
+              -e 's|\(header.path\) = .*|\1 = /app/include|'
+              -e 's|\(trans.path\) = .*|\1 = /app/share/qt5/translations|'
+              -e 's|\(qsci.path\) = .*|\1 = /app/share/qt5|'
+              -e 's|\(features.path\) = .*|\1 = /app/lib/qt5/mkspecs/features|'
+              -i Qt4Qt5/qscintilla.pro
+
+- name: json-c
+  sources:
+    - type: archive
+      url: https://github.com/json-c/json-c/archive/json-c-0.13.1-20180305.tar.gz
+      sha256: 5d867baeb7f540abe8f3265ac18ed7a24f91fe3c5f4fd99ac3caba0708511b90
+
+- shared-modules/intltool/intltool-0.51.json
+
+- name: libmypaint
+  config-opts:
+    - --disable-gegl
+    - --disable-introspection
+  sources:
+    - type: archive
+      url: https://github.com/mypaint/libmypaint/releases/download/v1.5.1/libmypaint-1.5.1.tar.xz
+      sha256: aef8150a0c84ce2ff6fb24de8d5ffc564845d006f8bad7ed84ee32ed1dd90c2b
+
+- name: quazip
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+  sources:
+    - type: archive
+      url: https://github.com/stachenov/quazip/archive/v0.8.1.tar.gz
+      sha256: 4fda4d4248e08015b5090d0369ef9e68bdc4475aa12494f7c0f6d79e43270d14
+    - type: shell
+      commands:
+        - sed -i 's/QUAZIP_LIB_VERSION_SUFFIX 5/QUAZIP_LIB_VERSION_SUFFIX ""/' CMakeLists.txt
+
+- name: 'x264'
+  config-opts:
+    - --disable-cli
+    - --enable-shared
+  sources:
+    - type: archive
+      url: https://code.videolan.org/videolan/x264/-/archive/296494a4011f58f32adc54304a2654627558c59a/x264-296494a4011f58f32adc54304a2654627558c59a.tar.bz2
+      sha256: cee6ae0628e2176dda590e26e1778443efad245a27543052f43ec1b5b96cf09a
+
+- name: ffmpeg
+  config-opts:
+    - --enable-rpath
+    - --enable-gpl
+    - --disable-static
+    - --enable-shared
+    - --disable-doc
+    - --disable-programs
+    - --enable-libopus
+    - --enable-libvpx
+    - --enable-libx264
+  cleanup: [/share]
+  sources:
+    - type: archive
+      url: https://www.ffmpeg.org/releases/ffmpeg-4.2.2.tar.xz
+      sha256: cb754255ab0ee2ea5f66f8850e1bd6ad5cac1cd855d0a2f4990fb8c668b0d29c
+
+- name: gperftools
+  post-install:
+    - mkdir /app/gperftools_root
+    - cp -r * /app/gperftools_root
+  cleanup: [/gperftools_root]
+  sources:
+    - type: archive
+      url: https://github.com/gperftools/gperftools/releases/download/gperftools-2.7/gperftools-2.7.tar.gz
+      sha256: 1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9
+    - type: patch
+      path: gperftools-enve-mod.patch
+
+- shared-modules/python2.7/python-2.7.json
```

Comment:
Just copy-paste it and add `cleanup: [*]`. No gain in centralized maintenance here as it's build-time dep.